pkg/oci/utils.go

```diff
@@ -37,6 +37,34 @@ var (
 
 	// BundlePathKey is the annotation key to fetch the OCI configuration file path.
 	BundlePathKey = "com.github.containers.virtcontainers.pkg.oci.bundle_path"
+
+	// ContainerTypeKey is the annotation key to fetch container type.
+	ContainerTypeKey = "com.github.containers.virtcontainers.pkg.oci.container_type"
+
+	// CRIOContainerTypeKey is the annotation key to fetch CRIO container type.
+	CRIOContainerTypeKey = "crio/container_type"
```

Comment:
Yes I agree it would  be even better that they export such values so that we don't need to define them ;)
As you've suggested, I will add those links to the comment !